src/codescene-interop.ts

```diff
@@ -130,17 +130,16 @@ function produceDiagnostic(severity: string, range: vscode.Range, message: strin
  * Excecutes the command for creating a code health rules template, and returns the result as a string.
  */
 export function codeHealthRulesJson(cliPath: string) {
-  const completedPromise = new Promise<string>((resolve, reject) => {
-    console.log('CodeScene: running "cs help code-health-rules-template"');
-    exec(`"${cliPath}" help code-health-rules-template`, (error, stdout, stderr) => {
-      if (error) {
-        console.error(`exec error: ${error}`);
-        reject(error);
-        return;
-      }
-      resolve(stdout);
-    });
-  });
-
-  return completedPromise;
+  console.log('CodeScene: running "cs help code-health-rules-template"');
+  const command: string = `"${cliPath}" help code-health-rules-template`;
+  return execAndLog(command, '"cs help code-health-rules-template"');
 }
+
+/**
+ * Executes the command for signing a payload, and returns the resulting signature as a string.
+ */
+export async function sign(cliPath: string, payload: string) {
+  console.log('CodeScene: running "cs sign" on ' + payload);
+  const command: string = `"${cliPath}" sign`;
+  return await execWithInput(command, "", payload);
```

Comment:
Very small thing: I would inline `command` here.

Reply:
My JS skills are lacking but I remember reading about `return await` being only useful inside try-catch blocks.
E.g. here: https://blog.battlefy.com/when-to-use-return-vs-await-vs-return-await-vs-nothing
Is it something that we want here?